addon/mixins/ajax-request.js

```diff
@@ -39,11 +39,24 @@ const {
 } = Ember;
 const JSONAPIContentType = 'application/vnd.api+json';
 
+// Header names are case-insensitive
+// https://www.w3.org/Protocols/rfc2616/rfc2616.html
+function retrieveHeader(headers, name) {
+  var found = null;
+  Object.keys(headers).forEach((key) => {
```

Comment:
This can be simplified using `.find`

```js
return Object.keys(headers).find((key) => {
  const value = headers[key];
  return value.toLowerCase() === name.toLowerCase();
});
```

The `.find` is sort of like `.filter`, which returns an array of only the elements in the original array for which the callback returns `true`.  However, it's different in that it only returns the first element in the array that matches, which is what it seems like you're going for here.